Source/WTF/wtf/PlatformHave.h

```diff
@@ -1625,7 +1625,8 @@
 
 #if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000) \
     || (PLATFORM(MACCATALYST) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
-    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)
+    || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000) \
```

Comment:
No need for iOS version check any more, we are not building anything older.

Reply:
I'll remove those in a follow up change. As there're quite a few of those like that in that file.